tests/infra_test.go

```diff
@@ -863,9 +863,9 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", func() {
 					req, _ := http.NewRequest("GET", metricsURL, nil)
 					resp, err := client.Do(req)
 					if err != nil {
-						fmt.Fprintf(GinkgoWriter, "client: request: %v #%d: %v\n", req, ix, err) // troubleshooting helper
+						logToGinkgoWritter("client: request: %v #%d: %v\n", req, ix, err) // troubleshooting helper
 					} else {
-						resp.Body.Close()
+						_ = resp.Body.Close()
```

Comment:
Why aren't we using `Expect`s here? ginkgo has a way of running expectations within goroutines iiuc

Reply:
In this particular instance, I think you're right - I could have used an `Expect` .

I'll try adding it.